api/src/routes/ical.ts

```diff
@@ -52,12 +54,15 @@ export function icalRoute(pool: Pool): Handler {
     const ctfs = await getCtfs();
 
     for (const ctf of ctfs) {
+      const ctftime_id = ctf.ctftime_url?.replace(/\/$/, "").split("/").at(-1);
       cal.createEvent({
+        id: `${ctf.id}:${ctftime_id || "no-ctftime"}@${config.pad.domain ||  "ctfnote"}`,
         start: ctf.start_time,
         end: ctf.end_time,
         description: ctf.description,
         summary: ctf.title,
         url: ctf.ctf_url,
+        attachments: [ctf.ctftime_url],
```

Comment:
@Minei3oat, I did a test on your code and I found that if you don't configure a `ctftime_url` then it will include a `ATTACH:null`. This is not according to the spec. Can you conditionally fill this array or give an empty array as attachments in case `ctf.ctftime_url` is not configured?

Reply:
I changed it to an empty array which results in no `ATTACH` line in the event.